<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сборка отчёта</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_stylesSetters">
      <item>
       <widget class="QCheckBox" name="checkBox_showLeft">
        <property name="text">
         <string>Окно сборки набора</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reprint">
        <property name="text">
         <string>Допечатка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_urdent">
        <property name="text">
         <string>Срочное</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_info">
        <property name="text">
         <string>Инфо</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clearStyle">
        <property name="text">
         <string>Очистить стиль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_join">
        <property name="text">
         <string>Объединить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="leftBox">
        <property name="title">
         <string>Набор</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QTableView" name="tableView_all"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_moveTop">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>⤒</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_moveUp">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>↑</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_moveDown">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>↓</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_moveBottom">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>⤓</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="CustomLineEdit" name="lineEdit_addNote"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_insertNote">
             <property name="text">
              <string>Вставить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_allButtons">
           <item>
            <widget class="QPushButton" name="pushButton_allExport">
             <property name="text">
              <string>Экспорт</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_allDelete">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_allClear">
             <property name="text">
              <string>Очистить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="rightBox">
        <property name="title">
         <string>Файл</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_fileSelector">
           <item>
            <widget class="QPushButton" name="pushButton_selectFile">
             <property name="text">
              <string>Обзор</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_inFile"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_reload">
             <property name="text">
              <string>Обновить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_presets" stretch="0,0,0,0,0,0,0,0,0">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Пресеты:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_1">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_3">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_4">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_5">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_6">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_7">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_savePreset">
             <property name="text">
              <string>Сохранить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView_current"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_curButtons">
           <item>
            <widget class="QPushButton" name="pushButton_curAdd">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_6">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_curExport">
             <property name="text">
              <string>Экспорт</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_curDelete">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_curClear">
             <property name="text">
              <string>Очистить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_appCfg"/>
    <addaction name="action_profileCfg"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_profileCfg">
   <property name="text">
    <string>Профиль</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_appCfg">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
